build/SetEnv.tasks

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+	<UsingTask  TaskName="SetEnv"
+                TaskFactory="CodeTaskFactory"
+                AssemblyFile="$(MSBuildToolsPath)\Microsoft.Build.Tasks.v12.0.dll">
+		<ParameterGroup>
+			<Name ParameterType="System.String" Required="true" />
+			<Value ParameterType="System.String" Required="true" />
+		</ParameterGroup>
+		<Task>
+			<Reference Include="System" />
+			<Code Type="Fragment" Language="cs">
+				<![CDATA[        
+					 Environment.SetEnvironmentVariable(Name, Value);
+					 
+					 Log.LogMessage (
+						"Environment varible is set, name={0}, value:{1}", 
+                        Name, 
+						Value);
+              ]]>
+			</Code>
+		</Task>
+	</UsingTask>
+
+	<UsingTask  TaskName="DisplayEnvVariables"
```

Comment:
What is the value of maintaining this task in addition to the built-in diagnostics logging of MSBuild, which already lists all project properties, including environment variables?